lib/themes/dosomething/paraneue_dosomething/scss/content/_win-module.scss

```diff
@@ -1,13 +1,11 @@
 .win-module {
   .win-module__progress {
     margin-bottom: $base-spacing;
-    background-image: asset-url("images/WinBackgroundS.svg");
```

Comment:
I don't think so if this is winning from a hot module. we still have the code for hot module, but we aren't using it anymoar
but there's a win state on campaigns... dunno which this is.